tests/src/python/test_qgsproject.py

```diff
@@ -1383,31 +1381,31 @@ def testGpkgDirtyingWhenRemovedFromStorage(self):
 
     def testMapScales(self):
         p = QgsProject()
-        self.assertFalse(p.mapScales())
-        self.assertFalse(p.useProjectScales())
+        self.assertFalse(p.viewSettings().mapScales())
```

Comment:
minor nit: having a vs = p.viewSettings() variable and reusing it afterwards would make code more compact